src/excercise7.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Created by ttn on 28/12/16.
+ */
+/**
+ * Create a User Registration form with fields ( FirstName, LastName, EmailId, Contact Number, Password and Confirm password).
+ * Form should have a submit button. Apply form validation to incorporate following rules.
+ *
+ */
+
+
+import React from 'react'
+
+export default class UserRegisration extends React.Component {
+
+  constructor(props){
+    super(props)
+    this.state = {
+      firstNameTxt : "",
```

Comment:
use self explanatory variable names. `firstNameTxt` is value of that input or the validation message?